vitest.workspace.ts

```diff
@@ -1 +1 @@
-export default ["apps/**", "packages/**"];
+export default ["apps/**", "packages/**", "load_tests/**"];
```

Comment:
**suggestion**: not required anymore.

I'm not sure we should add the `scripts/**` to the `vitest.workspace` since we do not have tests there
```suggestion
export default ["apps/**", "packages/**"];
```